files/build_templates/docker_image_ctl.j2

```diff
@@ -209,39 +238,52 @@ start() {
         {%- endif %}
         if [ x"$DOCKERMOUNT" == x"$HWSKU" ]; then
             {%- if docker_container_name == "database" %}
-            echo "Starting existing {{docker_container_name}} container"
+            echo "Starting existing {{docker_container_name}}$DEV container"
             {%- else %}
-            echo "Starting existing {{docker_container_name}} container with HWSKU $HWSKU"
+            echo "Starting existing {{docker_container_name}}$DEV container with HWSKU $HWSKU"
             {%- endif %}
             preStartAction
-            docker start {{docker_container_name}}
-            CURRENT_HOSTNAME="$(docker exec {{docker_container_name}} hostname)"
+            docker start {{docker_container_name}}$DEV
+            postStartAction
+            CURRENT_HOSTNAME="$(docker exec {{docker_container_name}}$DEV hostname)"
             if [ x"$HOSTNAME" != x"$CURRENT_HOSTNAME" ]; then
                 updateHostName "$HOSTNAME"
             fi
-            postStartAction
+            updateHostName "$HOSTNAME"
```

Comment:
Sorry for late comment.
Why it is needed to call updateHostName even though it is called in the condition right above?